easybuild/easyconfigs/r/Ruby-bundle/Ruby-bundle-2.3.4-Ruby-2.3.4.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'Bundle'
+
+name = 'Ruby-bundle'
+version = '2.3.4'
+versionsuffix = '-Ruby-%s' % version
+
+homepage = 'https://www.ruby-lang.org'
+description = """Ruby is a dynamic, open source programming language with 
+ a focus on simplicity and productivity. It has an elegant syntax that is 
+ natural to read and easy to write."""
```

Comment:
I think its self explanatory, Ruby provides a limited set of gems, purpose of this easyconfig is to add extra gems in Ruby-bundle. I followed the convention based on R-bundle. I don't know how to categorize bundle. It can also be called Ruby-extra or something else. The set of packages were installed to provide fpm capability so I can use this module to support --package with eb.